### PR TITLE
Review Introduction, Motivation

### PR DESCRIPTION
**Pull request by vrde**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,19 +35,24 @@ striking similarity to JavaScript has provoked criticism, the accusation of
 having too relaxed design principles for transferring money securely being
 thrown around on the Internet frequently. And indeed, Solidity features odd
 design choices. To name a few: (1) It doesn't have under- and overflow
-protection, (2) interfaces are not enforced and (3) similar modifiers are
-easily confused as their naming is virtually indistinguishable.
+protection, (2) interfaces are not enforced, (3) similar modifiers are easily
+confused as their naming is virtually indistinguishable, (4) code reuse is
```
Comment (TimDaub) [inline README.md:39]: I find the sentence of (4) to complex to read. Would you mind splitting it e.g. into multiple sentences?

File: README.md [hunk 2]
```diff
@@ -35,19 +35,24 @@ striking similarity to JavaScript has provoked criticism, the accusation of
...
+implemented through multiple inheritance, a pattern known for it's complexity
+and corner cases and often avoided even by seasoned programmers, that can lead
+to some non trivial bugs [13] and (5) inline assembly is hard to write and,
+even more important, difficult to read, making the smart contract difficult to
```
Comment (TimDaub) [inline README.md]: Isn't it "more importantly"? Not sure.

File: README.md [hunk 3]
```diff
@@ -71,30 +76,32 @@ and contract owners. To begin with, we elaborate reasons for why it is
...
 functionality of a smart contract (transparency). (2) Nobody must be able to
 exercise full control over a contract (decentralization). (3) Everybody must be
 be allowed, without the incurrence of a cost, to innovate on any part of the
-infrastructure (permission-less). It is therefore, and for other miscellaneous
-reasons, a best practice in the ecosystem to open source smart contracts. For a
-smart contract attacker, however, this yields a positive side effect. Namely,
-the source code being easily accessible.
+infrastructure (permissionless). It is therefore, and for other miscellaneous
+reasons, a best practice in the ecosystem to open source smart contracts. While
+we recognize that *security through obscurity* is not a viable defence
```
Comment (TimDaub) [inline README.md:87]: N1

**Review by TimDaub — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (vrde) [reply README.md]: "most important" should [be correct](https://english.stackexchange.com/questions/229517/is-more-importantly-good-english).

Comment (TimDaub) [reply README.md]: ok

**Pull request merged**